sdk/cmake/msvc.cmake

```diff
@@ -321,13 +321,11 @@ function(generate_import_lib _libname _dllname _spec_file)
     if(MSVC_IDE)
         add_library(${_libname} STATIC EXCLUDE_FROM_ALL ${_asm_stubs_file}.obj)
         set_source_files_properties(${_asm_stubs_file}.obj PROPERTIES EXTERNAL_OBJECT TRUE)
-        set_target_properties(${_libname} PROPERTIES LINKER_LANGUAGE "C")
     else()
         # NOTE: as stub file and def file are generated in one pass, depending on one is like depending on the other
         add_library(${_libname} STATIC EXCLUDE_FROM_ALL ${_asm_stubs_file})
-        # set correct "link rule"
-        set_target_properties(${_libname} PROPERTIES LINKER_LANGUAGE "IMPLIB")
```

Comment:
If we're no longer using the IMPLIB language, you can remove these: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=_IMPLIB_

Reply:
it's still used by oldnames. Will fix